packages/at_secondary_server/config/config.yaml

```diff
@@ -158,4 +158,9 @@ enrollment:
   # The maximum number of requests allowed within the time window.
   maxRequestsPerTimeFrame: 5
   # The duration of the time window in hours.
-  timeFrameInHours: 1
\ No newline at end of file
+  timeFrameInHours: 1
+  # The threshold value for the delay interval in seconds.
+  # When the last delay in '_delayForInvalidOTPSeries' surpasses this threshold,
+  # the delay in responding for Invalid OTP is reset to 1 second to prevent
+  #  excessively long delay intervals.
```

Comment:
- The delay should not be reset if there continue to be enroll requests with invalid OTPs, but should stay at the maximum (55 seconds)
- To put it another way - the delay should only be reset if <some duration, e.g. 89 seconds> has passed since the last enroll request which had an invalid OTP

Reply:
Thanks @sitaram-kalluri. Please can you make changes now to enable all of these delay-testing tests to run much more quickly (order of tens of milliseconds, not tens of seconds). You'll need to adjust various aspects of the implementation to do so.

Reply:
PS The code comments above are no longer accurate, please update them to reflect the actual behaviour.